eng/packaging.targets

```diff
@@ -39,7 +39,7 @@
     <UseIntellisenseDocumentationFile Condition="'$(UseIntellisenseDocumentationFile)' == '' and Exists('$(LibIntellisenseDocumentationFilePath)')">true</UseIntellisenseDocumentationFile>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
+  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing' or '$(PreReleaseVersionLabel)' == 'mauipreview'">
     <!-- If no servicing version is set we need to default to 0 in order for dependency versions to
     be calculated propertly, if we don't set it to 0, we would get the dependency version using the
     product Patch Version -->
```

Comment:
You don't really need this for the branch as we should be using the "default" assembly version we have that uses `Major.Minor.0.Patch`.